content/docs/authorization/configuration/proxy-server/_index.md

```diff
@@ -24,30 +24,33 @@ A `storage` entity in CSM Authorization consists of the storage type (PowerFlex,
 
 ```yaml
 # RPM Deployment
-karavictl storage create --type powerflex --endpoint https://10.0.0.1 --system-id ${systemID} --user ${user} --password ${password} --array-insecure
+karavictl storage create --type powerflex --endpoint ${powerflexIP} --system-id ${systemID} --user ${user} --password ${password} --array-insecure
 
 # Helm Deployment
-karavictl storage create --type powerflex --endpoint https://10.0.0.1 --system-id ${systemID} --user ${user} --password ${password} --insecure --array-insecure --addr storage.csm-authorization.com:<ingress-nginx-controller-port>
+karavictl storage create --type powerflex --endpoint ${powerflexIP} --system-id ${systemID} --user ${user} --password ${password} --insecure --array-insecure --addr storage.csm-authorization.com:<ingress-nginx-controller-port>
 ```
 
 >__Note__:
 > - The `insecure` flag specifies to skip certificate validation when connecting to the CSM Authorization storage service.
 > - The `array-insecure` flag specifies to skip certificate validation when proxy-service connects to the backend storage array. Run `karavictl storage create --help` for help.
+> - The `powerflexIP` is the endpoint to your powerflex machine. You can find the `systemID` at the `https://<your_powerflex_ip_address>/dashboard/performance` near the `System` title.
+> - The `user` and `password` arguments are credentials to the powerflex UI. 
 
 ### Configuring Tenants
 
 A `tenant` is a Kubernetes cluster that a role will be bound to. For example, to create a tenant named `Finance`:
 
 ```yaml
 # RPM Deployment
-karavictl tenant create --name Finance --insecure --addr grpc.DNS-hostname:443
+karavictl tenant create --name Finance --insecure --addr grpc.<DNS-hostname>:443
```

Comment:
can you please check <> brackets in the local preview?

Reply:
![dns](https://user-images.githubusercontent.com/117346178/218783670-90172437-d030-4d2c-b174-35a3faf265d5.PNG)